docs/Examples/bpmn-examples/theming.md

```diff
@@ -10,3 +10,9 @@ grand_parent: "Examples"
 ---
 
 # Theming
```

Comment:
Should we not rather have one page which lists all examples? Since it would be easier then for the user to get an overview. Maybe only this: https://github.com/bpmn-io/bpmn-js-examples/README.md embedded

Let's dicuss

Reply:
We actually have it here: https://bpmn-io.github.io/docs.bpmn.io/examples/bpmn

Reply:
We also have the overview page for every section. This is a default feature by the Jekyll plugin that we're using.